runtime/testy/studentCode.py

```diff
@@ -9,10 +9,28 @@ def autonomous_main():
   pass
 
 def teleop_setup():
-  pass
+  uids = Robot._getAllUIDs()
```

Comment:
So the counterargument I have is that master is what we will very soon be shipping directly from. We don't want to ship the software with preexisting code in `student_code.py`  This `teleop_setup()` code is exactly the function that students would write to. If they for some reason happened to press play without upload, it would run this code.